resources/assets/less/bem/simple-form.less

```diff
@@ -118,6 +118,11 @@
       flex: 1;
     }
 
+    &--modal {
+      background-color: @osu-colour-b6;
+      padding: 5px;
```

Comment:
i set it to the text position align with the selects above
![](https://i.imgur.com/2Y9EvPJ.png) vs ![](https://i.imgur.com/StHrJOl.png)

... but it's so marginal it could go i guess :thinking: